tests/unit/components/InMoreLanguageSwitch.spec.ts

```diff
@@ -0,0 +1,42 @@
+import Vuex from 'vuex';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import InMoreLanguageSwitch from '@/components/InMoreLanguageSwitch.vue';
+import { createStore } from '@/store';
+import {
+	NS_USER,
+	NS_MESSAGES,
+} from '@/store/namespaces';
+import { MESSAGES_INIT } from '@/store/messages/mutationTypes';
+import { LANGUAGE_INIT } from '@/store/user/mutationTypes';
+import { mutation } from '@/store/util';
+
+const localVue = createLocalVue();
+localVue.use( Vuex );
+
+describe( 'InMoreLanguageSwitch.vue', () => {
```

Comment:
```suggestion
describe( 'InMoreLanguageSwitch', () => {
```

Reply:
We did this also in the other components